lib/mapping/CMapService.cpp

```diff
@@ -44,7 +44,7 @@ std::unique_ptr<CMap> CMapService::loadMap(const ui8 * buffer, int size, const s
 
 	//might be original campaign and require patch
 	getMapPatcher(name)->patchMapHeader(header);
-	header.release();
+	header.reset();
```

Comment:
Looks like header will be removed from map. Did you try to run game?

Reply:
Can we use some std::move magic

Reply:
Original code is... somewhat weird - it creates unique_ptr that owns data already owned by another unique_ptr.
Because of that, this unique_ptr needs to be `release`d before going out of scope.

Wrong, no changes needed here